packages/core/upload/admin/src/pages/App/tests/MediaLibrary.test.js

```diff
@@ -179,59 +164,39 @@ describe('Media library homepage', () => {
           canRead: false,
           canCreate: false,
         });
-
         renderML();
-
         expect(screen.queryByText('Sort by')).not.toBeInTheDocument();
       });
 
       [
-        ['Most recent uploads', 'createdAt:DESC'],
-        ['Oldest uploads', 'createdAt:ASC'],
-        ['Alphabetical order (A to Z)', 'name:ASC'],
-        ['Reverse alphabetical order (Z to A)', 'name:DESC'],
-        ['Most recent updates', 'updatedAt:DESC'],
-        ['Oldest updates', 'updatedAt:ASC'],
-      ].forEach(([label, sortKey]) => {
-        it(`modifies the URL with the according params: ${label} ${sortKey}`, async () => {
+        'createdAt:DESC',
+        'createdAt:ASC',
+        'name:ASC',
+        'name:DESC',
+        'updatedAt:DESC',
+        'updatedAt:ASC',
+      ].forEach(sortKey => {
+        it(`modifies the URL with the according params: ${sortKey}`, async () => {
           const setQueryMock = jest.fn();
           useQueryParams.mockReturnValueOnce([{ rawQuery: '', query: {} }, setQueryMock]);
 
           renderML();
 
           fireEvent.mouseDown(screen.getByText('Sort by'));
-          await waitFor(() => expect(screen.getByText(label)).toBeInTheDocument());
-          fireEvent.mouseDown(screen.getByText(label));
-          await waitFor(() => expect(screen.queryByText(label)).not.toBeInTheDocument());
+          await waitFor(() => expect(screen.getByText(sortKey)).toBeInTheDocument());
+          fireEvent.mouseDown(screen.getByText(sortKey));
+          await waitFor(() => expect(screen.queryByText(sortKey)).not.toBeInTheDocument());
 
           expect(setQueryMock).toBeCalledWith({ sort: sortKey });
         });
       });
     });
 
     describe('select all', () => {
-      it('is not visible if there are not folders and assets', () => {
-        useAssets.mockReturnValueOnce({
-          isLoading: false,
-          error: null,
-          data: {},
-        });
-        useFolders.mockReturnValueOnce({
-          data: [],
-          isLoading: false,
-          error: null,
-        });
-        renderML();
-
-        expect(
```

Comment:
Why did you remove that test?

Reply:
good question 🤔

Reply:
mmh I guess I thought this test didn't make sense - I will add it back and fix it (we want to check that select all checkbox isn't there not if There are no elements with the applied filters is or is not in the document)